### DISCUSSION TITLE
Invalid syntax?

### DISCUSSION THREAD

**Discussion by JRampager** [meltaxa/solariot#42] [Q&A] [answered]
Hi,

I used to run a "non AES" script to monitor my inverter but it got upgraded and my script stopped working so i thought id try this.

I am by no means skilled in the arts of Python and i ran into what is probably e newbie problem.
In solariot.py row 66, logging.info(f"Loaded config {config.model}"), i get "invalid syntax (<string>, line 66)pylint(syntax-error)"
If i for the sake of testing comment row 66 the error appears on the next row and so forth.

Any tips or advice would be appreciated.

Thanks!

BR,

/R

**Comment by meltaxa** [comment, marked as answer]
Seems you are running Solariot via pylint. Use python solariot.py instead.

**Reply by JRampager** [reply]
Hi,

Thanks for your reply. 
Sorry for not providing more info but i am getting the same when executing the script in the terminal (through Visual Studio Code and "local" terminal on the PI).
I guess i have messed something up when it comes to the "python environment" on the PI.
Ill set up a new environment and give it another go.
